RELEASING.adoc

```diff
@@ -1,35 +1,57 @@
 = Spring Cloud GCP release process
 
-The actual release process is triggered by the release admin and happens together with the rest of the Spring Cloud release.
-In order to simplify that process, here is a list of prerequisites to be performed before asking the release admin to run the release scripts.
+The actual release process is triggered by the release admin.
+As opposed to the 1.x releases, the 2.0 release happens independent of the Spring Cloud release train.
+To simplify the process, perform this list of prerequisites before asking the release admin to run the release scripts.
 
-. Ensure that modules that shouldn't be released have the `maven-deploy-plugin` configuration set to `<skip>true</skip>`.
+. Ensure that modules that shouldn't be released are not included in the `<modules>` block in the `release` profile of `pom.xml`.
 For GA releases this should be configured under the `central` Maven profile.
```

Comment:
Do we even have the `central` profile now?

Reply:
Yes - but I don't know why this module is disabled: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/master/spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/pom.xml#L38

Reply:
Ah! Great question! This is what we say in our reference documentation about it.
```
The Google Cloud Runtime Configuration service is in Beta status, and is only available in snapshot and milestone versions of the project. It’s also not available in the Spring Cloud GCP BOM, unlike other modules.
```
I think it's a good time to decide to either completely remove it, or make it part of our GA releases.

Reply:
Good to know - it is in fact being included in our RC releases right now: https://mvnrepository.com/artifact/com.google.cloud/spring-cloud-gcp-starter-config/2.0.0-RC1